src/components/roi/ROIAnnotations.tsx

```diff
@@ -1,10 +1,10 @@
 import { memo, useEffect, useMemo, useRef } from 'react';
+import { usePanZoomTransform } from 'react-roi/lib-esm/hooks/usePanZoom';
```

Comment:
I made a fix in react-roi so that this hook is exported from the librar.
```suggestion
import { usePanZoomTransform } from 'react-roi';
```